### DISCUSSION TITLE
Timeline callout

### DISCUSSION THREAD

**Discussion by HS-MJ** [Seniblue/Underwater#9] [Q&A] [answered]
Hello,
Can I ask how you get the timeline callout to work with stacked callouts? I just get a large space between the items:

```
> [!timeline] Title1
> some text

> [!timeline] Title2
> some more text
```

![image](https://github.com/user-attachments/assets/582b541c-8cff-4b1a-ae14-e772d9437e3a)

(*Love* the theme by the way!)

**Comment by Seniblue** [comment, marked as answer]
Thanks for the feedback 😁!

The space between the items disappears in reading mode (but not in live preview).
![image](https://github.com/user-attachments/assets/0ce34303-e324-40ea-b78f-a522633c932d)

I will see if I can fix this, but unfortunately I don't think it is possible.
